Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM liblouis/liblouis
```

Comment:
Why did you do this? The liblouis docker image should contain everything as far as I know.

Reply:
Chris, when pulling from Docker hub the Debian Jessie based liblouis:liblouis image, because in Dockerfile need install Liblouis UTDML related dependency packages, need doing an apt update command.
Apt update trying refreshing the package cache, but Jessie system awailable packages are already not downloadable.
So, when you trying building Linux based version with docker build -f Dockerfile -t liblouisutdml:linux . command, the builded image will be not created, because in Debian Jessie system not installable the package build dependencies.
So, future need update the liblouis:liblouis Docker Hub awailable image for stable Debian system to again working the old Docker file (with uses now with liblouis:liblouis image).
You understand now the problem?

Attila

Reply:
I forgot to wrote:
Dockerfile.win32 and Dockerfile.win64 are not affected in Liblouis UTDML, because already pulling factory level with always awailable debian:latest image, and apt cache update list are always downloadable, because debian:latest image name is referencing always the latest stable Debian version.

Attila

Reply:
Hi Attila, please do not resolve conversations before I looked at it. The problem with the liblouis docker image is that it is very old. Docker hub used to pull from the github repository but it no longer seems to do that. They only do that for paying customers now. I have to manually push the (locally built) image.

Reply:
I manually pushed an up-to-date image to docker hub just now. This makes the following work:

``` console
~/src/liblouisutdml$ docker build -f Dockerfile -t liblouis/liblouisutdml .
Step 1/5 : FROM liblouis/liblouis
 ---> 6840b6870b37
Step 2/5 : RUN apt-get update && apt-get install -y     libxml2-dev    && rm -rf /var/lib/apt/lists/*
 ---> Running in 193f5be4674a
Get:1 http://deb.debian.org/debian bullseye InRelease [116 kB]
Get:2 http://deb.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
Get:4 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
Get:5 http://deb.debian.org/debian-security bullseye-security/main amd64 Packages [245 kB]
Get:6 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.8 kB]
Fetched 8651 kB in 2s (4351 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
etc
```
So you should be able to drop the change to the Dockerfile

Reply:
Ok Chris, thanks the change.
Fine if I do a git interactive rebase and remove Dockerfile related commits and do a force-push operation?

The liblouis:liblouis image are fine now, uses Debian Bullseye but bullseye ship security updates, but I don't no the eol date.
When Liblouis 3.27 are released, again need manual update the liblouis:liblouis image with debian:latest based image? I don't notified Docker Hub related change (with you wrote the paying customers related change), me not have a Docker hub account so I not notified this change unfortunately.
What your openion?

Attila

Reply:
> Fine if I do a git interactive rebase and remove Dockerfile related commits and do a force-push operation?

That should be fine, yes

> When Liblouis 3.27 are released, again need manual update the liblouis:liblouis image with debian:latest based image? 

Yes, a manual push is required now if we want the image updated. I was going to add an issue to liblouis about adding a git hub action to automate this.

> I don't notified Docker Hub related change (with you wrote the paying customers related change), me not have a Docker hub account so I not notified this change unfortunately.

Yes they sent a mail ages ago, but I did not want to pay to have up-to-date images

Reply:
Chris, I resolve the Dockerfile related sub topic part?
The you reviewed Dockerfile related change are already outdated, final yesterday because you pushed an uptodate Liblouis:liblouis image, I removed this change related commits before force-push operation.
If you have possibility, feel free resolve this sub topic discussion, or drop me a short commit and I press the resolve button with Dockerfile related review part.

Attila

Reply:
Ah, OK, Attila the Dockerfile issue is resolved. Thanks